### PR TITLE
Kueue verify with 1.17

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
https://github.com/golangci/golangci-lint/pull/2438

**Comment by ArangoGutierrez** [comment]
> /lgtm /approve

looks like is back online , but lgtm is not set

**Comment by ArangoGutierrez** [comment]
> @ArangoGutierrez: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> 
> Test name	Commit	Details	Required	Rerun command
> pull-test-infra-verify-lint	[a26205e](https://github.com/kubernetes/test-infra/commit/a26205e6b883db32955b09763e29e622a3ce4ee3)	[link](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/25657/pull-test-infra-verify-lint/1504180061878620160)	true	`/test pull-test-infra-verify-lint`
> [Full PR test history](https://prow.k8s.io/pr-history?org=kubernetes&repo=test-infra&pr=25657). [Your PR dashboard](https://prow.k8s.io/pr?query=is%3Apr%20state%3Aopen%20author%3AArangoGutierrez). Please help us cut down on flakes by [linking to](https://git.k8s.io/community/contributors/devel/sig-testing/flaky-tests.md#filing-issues-for-flaky-tests) an [open issue](https://github.com/kubernetes/test-infra/issues?q=is:issue+is:open) when you hit one in your PR.

yamllint was not happy with the comment

File: config/jobs/kubernetes-sigs/kueue/kueue-presubmits.yaml [hunk 1]
```diff
@@ -42,8 +42,8 @@ presubmits:
       testgrid-tab-name: pull-kueue-verify-main
       description: "Run kueue verify checks"
     spec:
-      containers:
-      - image: golang:1.18
+      containers:
```
Comment (ArangoGutierrez) [inline config/jobs/kubernetes-sigs/kueue/kueue-presubmits.yaml:45]: this trailing white space -> https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/25657/pull-test-infra-verify-lint/1504183906134396928

**Comment by ArangoGutierrez** [comment]
Ready for review

**Comment by ahg-g** [comment]
what about the other two? pull-kueue-test-unit-main and pull-kueue-test-integration-main; shouldn't we revert that too? I think this is broke presubmit; I am getting https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kueue/123/pull-kueue-test-integration-main/1504281421739462656 on https://github.com/kubernetes-sigs/kueue/pull/123

**Pull request merged**